Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Save-Word-document/Save-Word-document/Controllers/HomeController.cs

```diff
@@ -0,0 +1,301 @@
+            paragraph = table[2, 0].AddParagraph();
+            paragraph.ApplyStyle("Heading 1");
+            paragraph.ParagraphFormat.LineSpacing = 12f;
+
+            //Appends picture to the paragraph
+            FileStream image3 = new FileStream("Road-550-W.jpg", FileMode.Open, FileAccess.Read);
+            picture = paragraph.AppendPicture(image3);
+            picture.TextWrappingStyle = TextWrappingStyle.TopAndBottom;
+            picture.VerticalOrigin = VerticalOrigin.Paragraph;
+            picture.VerticalPosition = 3.75f;
+            picture.HorizontalOrigin = HorizontalOrigin.Column;
+            picture.HorizontalPosition = -5f;
+            picture.WidthScale = 92;
+            picture.HeightScale = 92;
+
+            //Appends paragraph
+            paragraph = table[2, 1].AddParagraph();
+            paragraph.ApplyStyle("Heading 1");
+            paragraph.ParagraphFormat.AfterSpacing = 0;
+            paragraph.ParagraphFormat.LineSpacing = 12f;
+            paragraph.AppendText("Road-150 ");
+
+            //Appends paragraph
+            paragraph = table[2, 1].AddParagraph();
+            paragraph.ParagraphFormat.AfterSpacing = 0;
+            paragraph.ParagraphFormat.LineSpacing = 12f;
+            paragraph.BreakCharacterFormat.FontSize = 12f;
+            paragraph.BreakCharacterFormat.FontName = "Times New Roman";
+            textRange = paragraph.AppendText("Product No: BK-R93R-44\r") as WTextRange;
+            textRange.CharacterFormat.FontSize = 12f;
+            textRange.CharacterFormat.FontName = "Times New Roman";
+            textRange = paragraph.AppendText("Size: 44\r") as WTextRange;
+            textRange.CharacterFormat.FontSize = 12f;
+            textRange.CharacterFormat.FontName = "Times New Roman";
+            textRange = paragraph.AppendText("Weight: 14\r") as WTextRange;
+            textRange.CharacterFormat.FontSize = 12f;
+            textRange.CharacterFormat.FontName = "Times New Roman";
+            textRange = paragraph.AppendText("Price: $3,578.27\r") as WTextRange;
+            textRange.CharacterFormat.FontSize = 12f;
+            textRange.CharacterFormat.FontName = "Times New Roman";
+
+            //Appends paragraph
+            section.AddParagraph();
+
+            //Saves the Word document to MemoryStream
+            MemoryStream stream = new MemoryStream();
+            document.Save(stream, FormatType.Docx);
+
+            //Upload the document to Dropbox
+            await UploadDocumentToDropBox(stream);
+
+            return Ok("Word document uploaded to DropBox Cloud Storage.");
+        }
+        /// <summary>
+        /// Upload file to Dropbox
+        /// </summary>
+        /// <param name="stream"></param>
+        /// <returns></returns>
+        public async Task<MemoryStream> UploadDocumentToDropBox(MemoryStream stream)
+        {
+            //Define the access token for authentication with the Dropbox API
+            var accessToken = "Access_Token";
```

Comment:
Add file path as separate variable as like download code

Reply:
Modified the code as suggested.